cursorless-talon-dev/src/spoken_form_test.py

```diff
@@ -72,7 +79,12 @@ def private_cursorless_spoken_form_test_mode(enable: bool):  # pyright: ignore [
 
         if enable:
             saved_modes = scope.get("mode")
-            saved_microphone = actions.sound.active_microphone()
+            if settings.get(
+                "user.cursorless_spoken_form_test_restore_default_microphone"
+            ):
+                saved_microphone = "System Default"
+            else:
+                saved_microphone = actions.sound.active_microphone()
```

Comment:
Per vc discussion, I think it's simpler to just have a string setting that the user can set to "system default" (or any other microphone), rather than a boolean that hardcodes a reference to "system default"